src/desktop/apps/article/ad_category.ts

```diff
@@ -0,0 +1,91 @@
+/**
+ * Parses article metadata to determine correct category type for Google DFP ad placements.
+ */
+
+interface ArticleProperties {
```

Comment:
Sorry but after digging a bit we already have an interface representing an Article types in reaction 
https://github.com/artsy/reaction/blob/master/src/Components/Publishing/https://github.com/artsy/reaction/blob/master/src/Components/Publishing/Typings.ts#L1-L7

And this whole class should really be in reaction as that's where all our logic really lives. Force is just a shell where we mount React components